include/libtorrent/error_code.hpp

```diff
@@ -92,6 +92,8 @@ namespace libtorrent {
 			invalid_escaped_string,
 			// Operation is not permitted since the session is shutting down
 			session_is_closing,
+			// The session_handle is not referring to a valid session_impl
+			invalid_session_handle,
```

Comment:
I've tried to keep the error codes stable across releases. That's why I've left gaps. Would you mind moving this to an empty slot further down?